src/evalflags.cpp

```diff
@@ -6,7 +6,7 @@
 
 namespace eppoclient {
 
-const std::string SDK_VERSION = getVersion();
+const char* const SDK_VERSION = getVersion();
```

Comment:
Global objects with destructors can cause undefined behavior at program exit due to indeterminate destruction order across translation units, which is what clang warns about with the `-Wexit-time-destructors` compile option. Built-in types like `char *` have no destructors, so this fixes the clang warning.

Reply:
TIL!